packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.base.tsx

```diff
@@ -87,7 +87,7 @@ export class SearchBoxBase extends React.Component<ISearchBoxProps, ISearchBoxSt
       id = this._fallbackId
     } = this.props;
     const { value, hasFocus } = this.state;
-    const placeholderValue = labelText === undefined ? placeholder : labelText;
+    const placeholderValue = placeholder || labelText;
```

Comment:
Could empty string be a valid value here?

Reply:
I made this change because `labelText` is the deprecated thing here, which was taking precedence if it was not `undefined`. I can change it so that this is `placeholder !== undefined ? placeholder : labelText` but I think we should still change the order since `placeholder` should take precedence over the deprecated `labelText`. What do you think about that approach @ecraig12345?